src/expression/comparison/index.ts

```diff
@@ -3,17 +3,26 @@
  * @module illogical/expression/comparison
  */
 
-import { Context, Evaluable, EvaluableType, Result } from '../../common/evaluable'
-import { isEvaluable } from '../../common/type-check'
+import {
+  Context,
+  Evaluable,
+  EvaluableType,
+  Result
+} from '../../common/evaluable'
+import { isEvaluable, isNumber, isString } from '../../common/type-check'
 import { Operand } from '../../operand'
 import { ExpressionInput } from '../../parser'
 import { Options } from '../../parser/options'
 
+export type ComparisonOptions = {
+  allowCrossTypeParsing: boolean;
+};
```

Comment:
should not be needed, https://github.com/briza-insurance/illogical/pull/137/files#r574117037

Reply:
9b2e2c0